test/gateway/index.js

```diff
@@ -27,7 +64,12 @@ describe('HTTP Gateway', () => {
     })
   })
 
-  describe('/ipfs/* route', () => {
+  describe('## interface tests', () => {
+    fs.readdirSync(path.join(__dirname, '/interface'))
+      .forEach((file) => require('./interface/' + file))
+  })
```

Comment:
@ya7ya why did you bring these tests back? These are designed for the API, not for the gateway.